eth/ethconfig/config.go

```diff
@@ -52,18 +52,19 @@ var DefaultConfig = NewDefaultConfig()
 
 func NewDefaultConfig() Config {
 	return Config{
-		NetworkId:             1,
-		TrieCleanCache:        512,
-		TrieDirtyCache:        256,
-		TrieDirtyCommitTarget: 20,
-		SnapshotCache:         256,
-		FilterLogCacheSize:    32,
-		Miner:                 miner.Config{},
-		TxPool:                core.DefaultTxPoolConfig,
-		RPCGasCap:             25000000,
-		RPCEVMTimeout:         5 * time.Second,
-		GPO:                   DefaultFullGPOConfig,
-		RPCTxFeeCap:           1,
+		NetworkId:                   1,
+		TrieCleanCache:              512,
+		TrieDirtyCache:              256,
+		TrieDirtyCommitTarget:       20,
+		SnapshotCache:               256,
+		IndexedFilterLogCacheSize:   32,
+		UnindexedFilterLogCacheSize: 32,
```

Comment:
We don't increase the default here because it could require more memory. This should be tuned by API providers.